README.md

```diff
@@ -1,31 +1,95 @@
+
 # Spark Design System
 
 Welcome to the Spark Design System, a product of [Quicken Loans](https://github.com/quickenloans).
 
-Spark is used to build the user interfaces for Quicken Loans Fintech products.
+Spark Design System is a collection of patterns used to create the user
+interface of Quicken Loans and Fintech products.
 
-## Using Spark
-Spark is currently in `alpha`. It is subject to changes of all sizes at any time and not suitable for production use. Please reach out to the [Spark team](mailto:sparkdesignsystem@quickenloans.com) for guidance.
+Spark's patterns are separated into two groups:
 
-### Feeling dangerous?
-Spark consists of 2 main packages which can be installed like so:
+* Spark-core
 
-* `npm install --save-dev @sparkdesignsystem/spark-core`
-  * Spark Core is the main package for the Spark Design System. This package contains the style and components that make up the basic interfaces for Quicken Loans Fintech products.
-* `npm install --save-dev @sparkdesignsystem/spark-extras`
-  * Spark Extras is a supplementary package that adds additional components that are not part of Spark Core. Many sites will use these components, but they aren't necessary for every Quicken Loans Fintech product.
+  Core consists of all basic patterns, the adoption of which will make the interface
+  of your application feel like a Quicken Loans, Fintech family of companies
+  app. Patterns in core have been identified as necessary to most if
+  not all applications.
 
-## Monorepo
+* Spark-extras
 
-Spark is managed as a monorepo. All of the Spark source code lives in a single repo, but is released as separate packages using [Lerna](https://github.com/lerna/lerna).
+  Extras patterns are things that are not needed on all applications, but when they
+  are needed, its desirable to be consistant.
 
 ## Getting Started
-More information on contributing to Spark will be available in the future. For now, if you want to get Spark running locally, follow these steps:
+
+**Spark assumes your project is using [Sass](https://sass-lang.com/) for styling and
+also that your project has a Javascript build step that will transpile and polyfill appropriately
+for browser support.**
+
+### Spark Core
+
+* Install the npm package
+
+  `npm install --save-dev @sparkdesignsystem/spark-core`
+
+* Import the setup file in your sass build
+
+  `@import node_modules/@sparkdesignsystem/spark';`
+
+  This will bring all spark-core patterns into your build. The spark classes
+  are namespaced and should not affect any existing css.
+
+### Spark Extras
+
+* Install the npm package
+
+  `npm install --save-dev @sparkdesignsystem/spark-extras`
+
+* Import any of the spark extras patterns that you need
+
+  `@import node_modules/@sparkdesignsystem/spark-extras/<pattern-name>/<pattern-name>';`
+
+  The available patterns as of the current release are:
```

Comment:
Maybe easier to just put a link to the package here instead of having to maintain a list.